libs/blocks/media/media.css

```diff
@@ -0,0 +1,218 @@
+.media {
+  margin: 0 auto;
+  position: relative;
+}
+
+.media.dark {
+  color: #fff;
+  background-color: #2c2c2c;
+}
+
+.media .background {
+  position: absolute;
+  width: 100%;
+  height: 100%;
+  object-fit: cover;
+}
+
+.media .background picture {
+  display: block;
+  position: absolute;
+  left: 0;
+  right: 0;
+  top: 0;
+  bottom: 0;
+}
+
+.media .background img {
+  object-fit: cover;
+  height: 100%;
+  width: 100%;
+}
+
+.media > .foreground .media-row {
+    align-items: center;
+    display: flex;
+    flex-direction: column;
+    gap:  var(--spacing-s);
+    justify-content: stretch;
+    margin: 0 auto;
+    padding: var(--spacing-s) 0;
+    position: relative;
+}
+
+.media.media-reverse-mobile .foreground .media-row {
+    flex-direction: column-reverse;
+}
+
+.media > .foreground .media-row > div {
+    width: 100%;
+}
+
+.media .text .detail-M,
+.media .text .heading-M,
+.media .text .heading-XS,
+.media .text .heading-S{
+  margin-bottom: var(--spacing-xxs);
+}
+
+.media .text .detail-L,
+.media .text .heading-L,
+.media .text .heading-XL,
+.media .text .body-S,
+.media .text .body-M {
+  margin-bottom: var(--spacing-xs);
+}
+
+.media .text .persona-area {
+  margin: 0 0 var(--spacing-xxs) 0;
+}
+
+.media .text .action-area {
+  display: flex;
+  margin: 0;
+  gap: var(--spacing-xs);
+  align-items: flex-start;
+}
+
+.text .icon-area {
+  margin: 0 0 var(--spacing-s) 0;
+  gap: var(--spacing-xs);
+  display: grid;
+  grid-template-columns: 1fr;
+}
+
+.text .icon {
+  align-items: center;
+  color: var(--link-color);
+  display: inline-flex;
+  font-size: var(--type-body-s-size);
+  font-weight: 700;
+  line-height: var(--type-body-l-size);
+  margin: 0;
+  padding: 0;
+  text-decoration: none;
+  text-transform: capitalize;
+}
+
+.text a.icon:hover {
+  text-decoration: underline;
+  color: var(--link-hover-color);
+}
+
+.text .icon img{
+  padding-right: var(--spacing-xxs);
+}
+
+.dark .text .icon,
+.dark .text .icon:hover{
```

Comment:
Why no change of color on hover?

Reply:
Good eye. I spoke w/ John on these dark.links and have aligned on this using the spectrum direction w/ lighter blue etc.